src/engine/Match_env.ml

```diff
@@ -31,8 +31,8 @@ module PM = Pattern_match
  *)
 type pattern_id = Xpattern.pattern_id
 
-(* !This hash table uses the Hashtbl.find_all property! *)
-type id_to_match_results = (pattern_id, Pattern_match.t) Hashtbl.t
+(* !This hash table uses the Hashtbl_.get_stack property! *)
```

Comment:
I don't think you need to put those comments anymore; Previously, I was adding such as comment because
the signature of the hashtbl does not indicate whether the hashtbl should be interpreted as a set of values
or one value. Now that the value type contains list ref, it's clear we're using a set of values for each keys.

Reply:
thanks. I was wondering.